src/mon/MonMap.cc

```diff
@@ -438,7 +438,7 @@ int MonMap::init_with_monmap(const std::string& monmap, std::ostream& errout)
   return r;
 }
 
-int MonMap::init_with_config_file(const ConfigProxy& conf,
+int MonMap::init_with_config_file(const ConfigReader& conf,
```

Comment:
could avoid this change if we want to specialize the compilation unit for seastar anyway -- we could conditionalize the `#include` the appropriate `config_proxy.h` on `WITH_SEASTAR` anyway.